src/Core_Command.php

```diff
@@ -152,8 +155,20 @@ public function download( $args, $assoc_args ) {
 
 		$locale       = Utils\get_flag_value( $assoc_args, 'locale', 'en_US' );
 		$skip_content = Utils\get_flag_value( $assoc_args, 'skip-content' );
+		$from_url     = false;
```

Comment:
We've established a convention for WP-CLI that we use `list` to fetch the positional arguments in one go.
```suggestion

		list( $download_url ) = $args;
		$from_url             = ! empty( $download_url );
```

Reply:
@schlessera the current build failure is due to this suggestion, `list( $var ) = $array` needs to be paired with a length check:

```
PHP Notice:  Undefined offset: 0 in src/Core_Command.php on line 159
```

Since this positional argument is optional, I'm not sure the `list()` pattern should apply here, but I'll wait for your suggestion before making a change.